test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -765,6 +765,12 @@ type_spec name alter = Test.group name <|
     Test.specify "should return a vector containing only unique elements up to some criteria" <|
         alter [Pair.new 1 "a", Pair.new 2 "b", Pair.new 1 "c"] . distinct (on = _.first) . should_equal [Pair.new 1 "a", Pair.new 2 "b"]
 
+    Test.specify "should be able to sort a heterogenous vector" <|
+        arr = [ 1, 1.3, "hi", Date.today, Date_Time.now, [ 0 ] ]
+        (arr.map .to_text . sort) . should_equal (arr.sort on=(.to_text) . map .to_text)
+        (arr.map .to_text . sort) . should_equal (arr.sort on=(_.to_text) . map .to_text)
+        (arr.map .to_text . sort) . should_equal (arr.sort on=(x-> x.to_text) . map .to_text)
```

Comment:
nitpick on tests style:

I'd flip the order of arguments here.

It's usually `tested_feature . should_equal expected_result`.

We are testing here the `on=`, so IMO it should be on the left-hand-side.

Reply:
Done in 653da76f3c8c0e34793f60ce1b12225658438193